projects/Mallard/package.json

```diff
@@ -68,7 +68,7 @@
         "react-native-iap": "^3.3.9",
         "react-native-image-resizer": "^1.0.1",
         "react-native-in-app-utils": "^6.0.2",
-        "react-native-inappbrowser-reborn": "^3.0.1",
```

Comment:
This might have been related to original sporadic bug itself @jimhunty that you were hitting. Do you have more details on what was not working? The browser not opening at all, or the sign-in failing with a "Sign-in cancelled" message? If the later, then it's most likely the bug that this PR precisely fixes. If the first, then I haven't seen any problem using `openAuth`.

Reply:
Nope unfortunately i dont. Just a double check. If everything is ok, LGTM.